<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyLoad GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/resources.qrc">
    <normaloff>:/icons/icons/logo.png</normaloff>:/icons/icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="downloadsTab">
       <attribute name="title">
        <string>Downloads</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="downloads">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeWidget" name="settingsTree">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="settingsArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="settings">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>615</width>
             <height>390</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="logBox">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPyLoad_Gui">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnectionManager"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionAddPackage"/>
     <addaction name="actionAddLinks"/>
     <addaction name="actionAddContainer"/>
     <addaction name="separator"/>
     <addaction name="actionAddAccount"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWiki"/>
    <addaction name="actionForum"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPyLoad_Gui"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionClipboard"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/images/toolbar_start.png</normaloff>:/images/images/toolbar_start.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="toolTip">
    <string>pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/images/toolbar_stop.png</normaloff>:/images/images/toolbar_stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/images/toolbar_add.png</normaloff>:/images/images/toolbar_add.png</iconset>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionClipboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/resources.qrc">
     <normaloff>:/images/images/clipboard.png</normaloff>:/images/images/clipboard.png</iconset>
   </property>
   <property name="text">
    <string>clipboard</string>
   </property>
  </action>
  <action name="actionConnectionManager">
   <property name="text">
    <string>Connection Manager</string>
   </property>
  </action>
  <action name="actionAddPackage">
   <property name="text">
    <string>Package</string>
   </property>
  </action>
  <action name="actionAddLinks">
   <property name="text">
    <string>Links</string>
   </property>
  </action>
  <action name="actionAddContainer">
   <property name="text">
    <string>Container</string>
   </property>
  </action>
  <action name="actionAddAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>GUI Settings</string>
   </property>
   <property name="toolTip">
    <string>GUI Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
